src/api/config/routes/webui_routes.rb

```diff
@@ -349,6 +349,8 @@
 
       resources :tokens, controller: 'webui/users/tokens' do
         resources :workflow_runs, only: [:index, :show], controller: 'webui/workflow_runs'
+        resources :users, only: [:index, :create, :destroy], controller: 'webui/users/tokens/users'
+        resources :groups, only: [:create, :destroy], controller: 'webui/users/tokens/groups'
```

Comment:
you need `constraints: cons` somewhere for those routes for users with a `.` in their username.

Reply:
done